app/src/main/res/values/dimens.xml

```diff
@@ -303,6 +303,11 @@
     <dimen name="weight_1">1</dimen>
     <dimen name="weight_2">2</dimen>
 
+    <dimen name="barometer_margin_top">54dp</dimen>
+
+    <dimen name="graph_editor_absoluteY">0dp</dimen>
```

Comment:
@yatri1609 Please remove these dimensions, I don't think they are used in the layout anymore

Reply:
Yes I'll remove it